### DISCUSSION TITLE
How to remove unused docker images

### DISCUSSION THREAD

**Discussion by vawaver** [nextcloud/all-in-one#1431] [Questions] [answered]
I would like to know whether it is possible to remove old unused docker images after upgrading to the newest version of AIO?
Or I must remove the images manually for example via Portainer?
Thank you.

**Comment by Zoey2936** [comment, marked as answer]
If you want, you can remove old unused and untagged images
But they also get automatically removed: https://github.com/nextcloud/all-in-one/discussions/800#discussioncomment-2900854
